api/db/migrations/trip/20191125100001_create_export_view.up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE MATERIALIZED VIEW trip.export AS (
+  SELECT
+    cpp.operator_id as operator_id,
+    tis.territory_id as start_territory_id,
+    tie.territory_id as end_territory_id,
+    cpp.acquisition_id::varchar as journey_id,
+    cpp.trip_id as trip_id,
+    cpp.datetime as journey_start_datetime,
+    trunc(ST_X(cpp.start_position::geometry)::numeric, cis.density::int) as journey_start_lat,
+    trunc(ST_Y(cpp.start_position::geometry)::numeric, cis.density::int) as journey_start_lon,
+    cpp.start_insee as journey_start_insee,
+    cis.postcodes[0] as journey_start_postalcode,
+    cis.town as journey_start_town,
+    null as journey_start_EPCI, -- TODO
+    cis.country as journey_start_country,
+    (cpp.datetime + (cpp.duration || ' seconds')::interval) as journey_end_datetime,
+    trunc(ST_X(cpp.end_position::geometry)::numeric, cie.density::int) as journey_end_lat,
+    trunc(ST_Y(cpp.end_position::geometry)::numeric, cie.density::int) as journey_end_lon,
+    cpp.end_insee as journey_end_insee,
+    cie.postcodes[0] journey_end_postalcode,
+    cie.town as journey_end_town,
+    null as journey_end_EPCI,  -- TODO
+    cie.country as journey_end_country,
+    (CASE WHEN cpp.distance <> null THEN cpp.distance ELSE (cpp.meta::json->'calc_distance')::text::int END) as journey_distance,
```

Comment:
```suggestion
    (CASE WHEN cpp.distance <> null THEN cpp.distance ELSE (cpp.meta::json->>'calc_distance')::int END) as journey_distance,
```
L'opérateur `->>` renvoie du ::text et l'opérateur `->` renvoie un objet.